workspace.json

```diff
@@ -1540,7 +1540,7 @@
               "apps/vyper/src/styles.css"
             ],
             "scripts": [],
-            "webpackConfig": "@nrwl/react/plugins/webpack"
+            "webpackConfig": "apps/vyper/webpack.config.js"
```

Comment:
Is this change necessary for this PR?

Reply:
Yes, vyper plugin was failing e2e. @bunsenstraat helped fix the issue which had to do with webpack.